ipamutils/utils.go

```diff
@@ -48,3 +60,75 @@ func initGranularPredefinedNetworks() []*net.IPNet {
 	}
 	return pl
 }
+
+// InitAddressPools allows to initialize the local and global scope predefined
+// address pools to the desired values. It fails is invalid input is passed
+// or if the predefined pools were already initialized.
+func InitAddressPools(list []*PredefinedPools) error {
+	var (
+		localPools []*net.IPNet
```

Comment:
instead of declaring an empty slice, please allocate it already to the size of list, that seems to be the expected length. This will allow a better memory management, else there would be tons of copies.